react/components/ProductAvailability.tsx

```diff
@@ -1,48 +1,53 @@
-import React, { FunctionComponent, memo } from 'react'
-import classNames from 'classnames'
-
-import styles from '../styles.css'
+import React, { memo } from 'react'
+import type { ProductTypes } from 'vtex.product-context'
 
 import LowStock from './LowStock'
 import HighStock from './HighStock'
+import Container from './Container'
 
 interface Props {
   threshold: number
-  lowStockMessage: string
-  highStockMessage: string
-  availableQuantity: number | null | undefined
-}
-
-const Container: FunctionComponent = ({ children }) => {
-  return (
-    <div className={classNames(styles.container, 'flex pv2')}>
-      {children}
-    </div>
-  )
+  lowStockMessage?: string
+  highStockMessage?: string
+  availableQuantity?: ProductTypes.CommercialOffer['AvailableQuantity']
 }
 
-const ProductAvailability: FunctionComponent<Props> = ({ threshold, lowStockMessage, highStockMessage, availableQuantity }) => {
-  if (availableQuantity == null || availableQuantity < 1) {
+function ProductAvailability({
+  threshold,
+  lowStockMessage,
+  highStockMessage,
+  availableQuantity,
+}: Props) {
+  if (
+    availableQuantity == null ||
+    availableQuantity === undefined ||
```

Comment:
`availableQuantity == null` will be `false` if `availableQuantity` is `undefined`, so this check is redundant.

You could either remove it or change the first one to `availableQuantity === null`. Ooor, you could even do `!Boolean(availableQuantity)`

Reply:
The lint was angry about `!Boolean(availableQuantity)` so I just `!availableQuantity` 😄